remix-run/remix
# ScrollRestoration on RR 6.4

Update to `ScrollRestoration` to wrap the RR 6.4 internals

Testing:
* ~Needs to be run with changes from https://github.com/remix-run/react-router/pull/9664~
* Confirm client side scroll restoration works on back/forward button routing
* Confirm SSR restoration works on hard reloads
* Confirm `<ScrollRestoration getKey={(location, matches) => ...} />` works
* Confirm `<Link preventScrollReset>` works
